movie-showcase/src/routes/managing/+server.ts

```diff
@@ -0,0 +1,15 @@
+import { API_KEY } from '$env/static/private'
+
+import { json } from '@sveltejs/kit';
+import type { RequestHandler } from '@sveltejs/kit';
+
+export const POST: RequestHandler = async ({ request }) => {
+    const { title, year } = await request.json();
+    
+    const url = `https://www.omdbapi.com/?t=${title}&y=${year}&apikey=${API_KEY}`;
```

Comment:
Ideally, the API base endpoint should be configurable in the `.env` file, just like `API_KEY`.